test/Makefile

```diff
@@ -6,7 +6,7 @@ bins:
 fuzz/targets:
 	git clone --depth 1 https://github.com/radareorg/radare2-fuzztargets fuzz/targets
 
-RUNTEST=cd new && r2r -L
+RUNTEST=cd new && r2r
```

Comment:
`-L` is a valid argument for the new r2r.c implementation. What kind of r2r do you have locally? Node.js and V implementations have been killed recently.

Reply:
Ok sorry I did not notice. I made the changed.